manifests/base/jupyter-rocm-pytorch-notebook-imagestream.yaml

```
@@ -0,0 +1,30 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/blob/main/jupyter/rocm/pytorch"
+    opendatahub.io/notebook-image-name: "ROCm-PyTorch"
+    opendatahub.io/notebook-image-desc: "Jupyter ROCm optimized PyTorch notebook image for ODH notebooks."
+    opendatahub.io/notebook-image-order: "14"
+    opendatahub.io/recommended-accelerators: '["habana.ai/gaudi"]'
```

I think this should rather be:
```suggestion
    opendatahub.io/recommended-accelerators: '["amd.com/gpu"]'
```
Same for the other occurrence.